.github/workflows/phpstan.yml

```diff
@@ -8,21 +8,20 @@ on:
 
 jobs:
   phpstan:
-    name: Analyze PHP ${{ matrix.php-versions }} (${{ matrix.refs == 'pr-head' && github.head_ref || github.base_ref || github.ref_name }})
+    name: Analyze PHP (${{ matrix.refs == 'pr-head' && github.head_ref || github.base_ref || github.ref_name }})
     runs-on: [ubuntu-latest]
     strategy:
       fail-fast: false
       matrix:
         refs: ['target-branch', 'pr-head']
         exclude:
           - refs: ${{ github.event_name != 'pull_request' && 'pr-head' }}
-        php-versions: ['8.2', '8.3', '8.4']
```

Comment:
question: why did you remove the older php versions?

Reply:
PHPStan v2 now has a [phpVersion min / max setting](https://phpstan.org/config-reference#phpversion).

So we put in `.phpstan.dist.neon`:

```neon
parameters:
    phpVersion:
        min: 80200
        max: 80400
```

Which means PHPStan doesn't need to run so many times, and it should be much quicker.

Reply:
oh right that's very nice to run phpstan only once!

Reply:
Yup, well only twice now. One for the PR Head and one for the merge commit.